package.json

```diff
@@ -22,7 +22,7 @@
   },
   "repository": {
     "type": "git",
-    "url": "git://github.com/criso/fbgraph.git"
+    "url": "git://github.com/mgara/fbgraph.git"
```

Comment:
I think you should leave it as it was

Reply:
sure ... that's why I explicitly mentioned the commit : bde6d29  
And this commit is originated from my fork... but thanks for the comment.